tests/cpp/misc/libinfo_test.cc

```diff
@@ -30,4 +30,5 @@ using namespace mxnet::features;
 TEST(RuntimeTest, RuntimeTestAll) {
   EXPECT_EQ(EnumNames::names.size(), MAX_FEATURES);
   const auto& features = LibInfo::getInstance()->getFeatures();
+  EXPECT_FALSE(features.empty());
```

Comment:
Is this change related to this refactoring?

Reply:
no, fixes a warning.